warning: implicit conversion from 'int' to 'float'

```
/home/chiller/Desktop/git/ddnet/src/tools/map_replace_area.cpp:667:18: warning: implicit conversion from 'int' to 'float' changes value from 2147483647 to 2147483648 [-Wimplicit-const-int-float-conversion]                                                                                                                 
        return Value == INEXISTENT;                                                                                                                                                                                                                                                                                           
                     ~~ ^~~~~~~~~~                                                                                                                                                                                                                                                                                            
/home/chiller/Desktop/git/ddnet/src/tools/map_replace_area.cpp:8:20: note: expanded from macro 'INEXISTENT'                                                                                                                                                                                                                   
#define INEXISTENT INT_MAX                                                                                                                                                                                                                                                                                                    
                   ^~~~~~~                                                                                                                                                                                                                                                                                                    
/usr/lib/clang/13.0.1/include/limits.h:46:19: note: expanded from macro 'INT_MAX'                                                                                                                                                                                                                                             
#define INT_MAX   __INT_MAX__                                                                                                                                                                                                                                                                                                 
                  ^~~~~~~~~~~                                                                                                                                                                                                                                                                                                 
<built-in>:93:21: note: expanded from here                                                                                                                                                                                                                                                                                    
#define __INT_MAX__ 2147483647                                                                                                                                                                                                                                                                                                
                    ^~~~~~~~~~     
 ```

--- a/src/tools/map_replace_area.cpp
+++ b/src/tools/map_replace_area.cpp
@@ -1,5 +1,3 @@
-#include <algorithm>
-#include <limits>
 #include <base/logger.h>
 #include <base/system.h>
 #include <engine/shared/datafile.h>

